### PR TITLE
EASY-2741: Create schema for archeology filemetadata

### PR DESCRIPTION
**Pull request by lindareijnhoudt**
fixes EASY-2741

#### When applied it will
* Create schema for archeology filemetadata; 
* Import schema in files.xsd
* [x] Add all relevant fields 

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-fedora2vault                      | [PR#](PRlink)

### CODE REVIEW

File: lib/src/main/resources/bag/metadata/afm/2020/afm.xsd [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        <xs:annotation>
+            <xs:documentation xml:lang="dut"
+                >aanvullingen en specifieke instructies voor
+                (her)gebruik</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="case_quantity" substitutionGroup="dcterms:extent">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut">aantal waarnemingen</xs:documentation>
```
Comment (jo-pol) [inline lib/src/main/resources/bag/metadata/afm/2020/afm.xsd:40]: VG mentioned some misinterpretation in the past. Shouldn't we elaborate more?

File: lib/src/main/resources/bag/metadata/afm/2020/afm.xsd [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        <xs:annotation>
+            <xs:documentation xml:lang="dut">technische bestandsspecificatie</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="othmat_codebook" substitutionGroup="dcterms:requires">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut"
+                >exact bestandsnaam van het bijbehorende codeboek
+                (monsterlijst_code.txt)</xs:documentation>
```
Comment (jo-pol) [inline lib/src/main/resources/bag/metadata/afm/2020/afm.xsd:66]:
```suggestion
                (e.g. monsterlijst_code.txt)</xs:documentation>
```

File: lib/src/main/resources/bag/metadata/afm/afm.xsd [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:dcterms="http://purl.org/dc/terms/" elementFormDefault="qualified"
+    targetNamespace="http://easy.dans.knaw.nl/schemas/bag/metadata/afm/"> 
+    
+    <xs:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.w3.org/2001/03/xml.xsd"/>
+    <xs:import namespace="http://purl.org/dc/terms/" schemaLocation="https://dublincore.org/schemas/xmls/qdc/dcterms.xsd"/>
+    
+    <xs:element name="mapprojection" substitutionGroup="dcterms:description"/>
+    <xs:element name="analyticUnits" substitutionGroup="dcterms:description"/>
+    <xs:element name="notes" substitutionGroup="dcterms:description"/>
+    <xs:element name="software" substitutionGroup="dcterms:description"/>
```
Comment (jo-pol) [inline lib/src/main/resources/bag/metadata/afm/afm.xsd:11]: I don't understand why this one is a subset of the 2020 version.

File: lib/src/main/resources/bag/metadata/afm/2020/afm.xsd [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="case_quantity" substitutionGroup="dcterms:extent">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut">aantal waarnemingen</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="file_category" substitutionGroup="dcterms:subject">
```
Comment (jo-pol) [inline lib/src/main/resources/bag/metadata/afm/2020/afm.xsd:44]: So far I mapped this one to `<dcterms:type>`.

File: lib/src/main/resources/bag/metadata/afm/2020/afm.xsd (hunk 1 above)
Comment (lindareijnhoudt) [reply lib/src/main/resources/bag/metadata/afm/2020/afm.xsd:40]: that should be mentioned in the conversion-step in easy-fedora2vault

File: lib/src/main/resources/bag/metadata/afm/afm.xsd (hunk 3 above)
Comment (lindareijnhoudt) [reply lib/src/main/resources/bag/metadata/afm/afm.xsd]: fixed

**Pull request merged**